status-client-spec.md

```diff
@@ -18,56 +18,70 @@ This document consists of two parts. The first outlines the specifications that
-    - [Data sync](#data-sync-1)
-      - [Why is MVDS not used for public chats?](#why-is-mvds-not-used-for-public-chats)
-  - [Footnotes](#footnotes)
-  - [Appendix A: Security considerations](#appendix-a-security-considerations)
-    - [Scalability and UX](#scalability-and-ux)
-    - [Privacy](#privacy)
-    - [Spam resistance](#spam-resistance)
-    - [Censorship resistance](#censorship-resistance)
-  - [Acknowledgements](#acknowledgements)
+
+-   [Status Client Specification](#status-client-specification)
+    -   [Abstract](#abstract)
+    -   [Table of Contents](#table-of-contents)
+    -   [Introduction](#introduction)
+        -   [Protocol layers](#protocol-layers)
+        -   [Protobuf](#protobuf)
+    -   [Components](#components)
+        -   [P2P Overlay](#p2p-overlay)
+            -   [Node discovery and roles](#node-discovery-and-roles)
+            -   [Bootstrapping](#bootstrapping)
+            -   [Discovery](#discovery)
+            -   [Mobile nodes](#mobile-nodes)
+        -   [Transport privacy and Whisper usage](#transport-privacy-and-whisper-usage)
+        -   [Secure Transport](#secure-transport)
+        -   [Data Sync](#data-sync)
+        -   [Payloads and clients](#payloads-and-clients)
+        -   [BIPs and EIPs Standards support](#bips-and-eips-standards-support)
+    -   [Security Considerations](#security-considerations)
+    -   [Design Rationale](#design-rationale)
+        -   [P2P Overlay](#p2p-overlay-1)
+            -   [Why devp2p? Why not use libp2p?](#why-devp2p-why-not-use-libp2p)
+            -   [What about other RLPx subprotocols like LES, and Swarm?](#what-about-other-rlpx-subprotocols-like-les-and-swarm)
+            -   [Why do you use Whisper?](#why-do-you-use-whisper)
+            -   [I heard you were moving away from Whisper?](#i-heard-you-were-moving-away-from-whisper)
+            -   [Why is PoW for Whisper set so low?](#why-is-pow-for-whisper-set-so-low)
+            -   [Why do you not use Discovery v5 for node discovery?](#why-do-you-not-use-discovery-v5-for-node-discovery)
+            -   [I heard something about mailservers being trusted somehow?](#i-heard-something-about-mailservers-being-trusted-somehow)
+        -   [Data sync](#data-sync-1)
+            -   [Why is MVDS not used for public chats?](#why-is-mvds-not-used-for-public-chats)
+    -   [Footnotes](#footnotes)
+    -   [Appendix A: Security considerations](#appendix-a-security-considerations)
+        -   [Scalability and UX](#scalability-and-ux)
+        -   [Privacy](#privacy)
+        -   [Spam resistance](#spam-resistance)
+        -   [Censorship resistance](#censorship-resistance)
+    -   [Acknowledgements](#acknowledgements)
 
 ## Introduction
 
+The Status mobile app is a multifunction application. This document mainly describes the requirements for establishing connections and communicating between clients. The other features specific to the Status application can be found as follows:
```

Comment:
Sorry I don't like this section, it links to no-op work and the story it tells is weird. Document deals with way more than connections, and the main thing that are specified aren't liked. We can reword this into PR once other specs are further along. Removing.